src/js/outreach/entities-service/index.js

```diff
@@ -41,62 +41,6 @@ function getToken({ dob, actionId }) {
     });
 }
 
-function getPatientInfo({ actionId }) {
-  return fetcher(`/api/outreach?filter[action]=${ actionId }`, {
-    method: 'GET',
-  })
-    .then(handleJSON)
-    .then(({ data }) => {
-      return data;
-    });
-}
-
-function createVerificationCode({ patientId }) {
-  return fetcher(`/api/outreach/${ patientId }`, {
-    method: 'POST',
-    data: {},
-  })
-    .then(handleJSON);
-}
-
-function validateVerificationCode({ patientId, code }) {
-  const data = {
-    type: 'outreach',
-    attributes: {
-      patient_id: patientId,
-      otp: code,
-    },
-  };
-
-  return fetcher('/api/outreach/auth', {
-    method: 'POST',
-    data: JSON.stringify({ data }),
-  })
-    .then(handleJSON)
-    .then(({ data: { attributes } }) => {
-      Radio.request('auth', 'setToken', attributes.token);
-      return Promise.resolve(attributes.token);
-    });
-}
-
-function optInPostRequest({ inputData }) {
-  const data = {
-    type: 'outreach',
-    attributes: {
-      first_name: inputData.get('firstName'),
-      last_name: inputData.get('lastName'),
-      birth_date: inputData.get('dob'),
-      phone: inputData.get('phone'),
-    },
-  };
-
-  return fetcher('/api/outreach', {
-    method: 'POST',
-    data: JSON.stringify({ data }),
-  })
-    .then(handleJSON);
-}
-
 function postResponse({ formId, actionId, response }) {
```

Comment:
this post response can probably just stay here with the handler for `registerPatient`, but instead of exporting the functions we can use the `entiities` channel for these.

However this function could be simplified since we have the models..
```js
const Model from 'js/base/model';

const FormResponse = Model.extend({
  url() {
    return `/api/actions/${ this.get('_action') }/relationships/form-responses`;
  },
  saveAll() {
    const attrs = this.attributes;

    const relationships = {
      'form': this.toRelation(attrs._form, 'forms'),
      'action': this.toRelation(attrs._action, 'patient-actions'),
    };

    return this.save(attrs, { relationships }, { wait: true });
  },
});

function postResponse({ formId, actionId, response }) {
  const response = new FormResponse({ _action: actionId, _form: formId, response });
  return response.saveAll();
}
```

We can probably utilize the base model for the patient request as well.